src/domain/map/reducer.ts

```diff
@@ -78,6 +91,8 @@ const initialState: ReducerState = {
     [MAPTILES.ORTOKARTTA]: buildTilelayerState(MAPTILES.ORTOKARTTA, false),
     [MAPTILES.KANTAKARTTA]: buildTilelayerState(MAPTILES.KANTAKARTTA, true),
   },
+  hankeFilterStartDate: `${currentYear}-01-01`,
```

Comment:
Voisi olla luettavampi ja selkeämpi jos siirtää filtterin objektin sisään. Jos näitä filttereitä tulee tulevaisuudessa lisää, niin voi liikutella kokonaista objektia useamman muuttujan sijaan. 

```
hankeFilterStartDate: `${currentYear}-01-01`,
hankeFilterEndDate: `${currentYear + 1}-12-31`,
```

-->

```
hankeFilters: {
  startDate: `${currentYear}-01-01`,
  endDate: `${currentYear + 1}-12-31`
}
```